stable/ghost/templates/deployment.yaml

```diff
@@ -17,7 +17,8 @@ spec:
     spec:
       initContainers:
       - name: volume-permissions
-        image: busybox
+        image: "{{ .Values.initContainerImage }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
```

Comment:
@dhilipkumars Bitnami's internal test system relies on the current naming of keys in `values.yml` file. Changing this will break our internal build/test pipeline. This is not something we can change with immediate effect.

Also I do not think there is any such policy for ordering keys in `values.yaml`